app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import shutil
+from typing import Any
+from app.errors import (
+    IncorrectCommand,
+    IncorrectCopyCommand,
+    IncorrectSecondFilename,
+    FileNotExists
+)
+from app.validator import command_validator
+
+
+def copy_file(command: str) -> Any:
+    inner_command = command.split()
+    try:
+        command_validator(inner_command)
+    except IncorrectCommand:
+        print("Incorrect input of command in general")
+    except IncorrectCopyCommand:
+        print("Incorrect command for copying")
+    except FileNotExists:
+        print("The file to copy does not exist")
+    except IncorrectSecondFilename:
+        print("Input a new name for the file")
```

Comment:
It must do nothing in case the user is trying to copy file to file with the same name.
https://github.com/mate-academy/py-copy-file#:~:text=It%20must%20do%20nothing%20in%20case%20the%20user%20is%20trying%20to%20copy%20file%20to%20file%20with%20the%20same%20name.